Power Up Proto/src/org/usfirst/frc/team5493/robot/Robot.java

```diff
@@ -35,10 +43,11 @@
     
     public void robotInit() {
     	
-    	climber = new Climber();
+    	climber = new OneClimbyBoi();
     	cubeControls = new CubeControls();
+    	throwDaggersInBensEyes = new ThrowDaggersInBensEyes();
 		oi = new OI();
-//		solenoid = new Solenoid();
+//		throwDaggersInBensEyes = new ThrowDaggersInBensEyes();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)